plugins/scanners/askalono/src/main/kotlin/Askalono.kt

```diff
@@ -69,8 +57,75 @@ class Askalono internal constructor(name: String, private val wrapperConfig: Sca
         // askalono 0.2.0-beta.1
         output.removePrefix("askalono ")
 
+    override fun displayName() = "Askalono"
```

Comment:
Can this be omitted now with 768170547ac4ca1e43bf01a54d61c9a489fdb809? Similar for other scanners.

Reply:
The removal should have been done in the first commit of this PR already.